resources/views/events/aanmelden.blade.php

```diff
@@ -41,6 +41,18 @@
                         <div class="alert alert-danger">{{ $message }}</div>
                         @enderror
                     </div>
+                    <div>
+                        <label for="gender" class="col-sm-2 col-form-label">
+                            <span class="requiredStar">*</span>Geslacht
```

Comment:
Er is een verschil tussen geslacht en gender, geslacht is biologisch en gender is waarmee je je identificeert. Omdat niet iedereen zich identificeert met zijn geslacht kan je hier beter naar de gender vragen